osu.Game/Screens/OnlinePlay/Multiplayer/Match/Playlist/MultiplayerPlaylistTabControl.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Bindables;
+using osu.Framework.Graphics.UserInterface;
+using osu.Game.Graphics.UserInterface;
+
+namespace osu.Game.Screens.OnlinePlay.Multiplayer.Match.Playlist
+{
+    public class MultiplayerPlaylistTabControl : OsuTabControl<MultiplayerPlaylistDisplayMode>
+    {
+        public Bindable<int> queueListCount = new Bindable<int>(0);
```

Comment:
In general, bindables that receive value from external sources should be immutable (`IBindable...`) and readonly.

Also always instantiate the most derived version of any bindable to avoid potential issues from inheritance with base properties, i.e. use `BindableInt`/`BindableFloat`/`BindableDouble` instead of `Bindable<int>`/`Bindable<float>`/`Bindable<double>`:

And lastly, arguments that are equal to the default parameter values are redundant and will trigger CI.
```suggestion
        public readonly IBindable<int> queueListCount = new BindableInt();
```